src/vcf/vcf_v41.ragel

```diff
@@ -58,140 +58,140 @@
 
 
     ## A chromosome must be a string with no white-spaces or colons, and may be surronded by < > symbols (for contigs)
-    chrom_basic     = alnum ( alnum | punct - (":" | "<" | ">" | ",") )* ;
-    chrom_contig    = "<" chrom_basic ">" ;
+    chrom_basic     = alnum ( alnum | punct - (':' | '<' | '>' | ',') )* ;
+    chrom_contig    = '<' chrom_basic '>' ;
     chromosome      = chrom_basic | chrom_contig ;
 
     # File format line
-    fileformat_name = "VCFv4.1" >token_begin $token_middle %fileformat_end $err(fileformat_error) ;
-    fileformat      = "##fileformat=" fileformat_name ;
+    fileformat_name = 'VCFv4.1' >token_begin $token_middle %fileformat_end $err(fileformat_error) ;
+    fileformat      = '##fileformat=' fileformat_name ;
 
     # Meta-data
-    meta_typeid     = ( ( print - "=" )+ - ("ALT" | "FILTER" | "FORMAT" | "INFO" | "assembly" | "contig" | "SAMPLE" | "PEDIGREE" | "pedigreeDB") ) >token_begin @token_middle %meta_typeid_generic ;
-    meta_key        = ( (alnum | "." | "_" | "-" )+ - (punct)+ ) >token_begin @token_middle %meta_key_generic;
+    meta_typeid     = ( ( print - '=' )+ - ('ALT' | 'FILTER' | 'FORMAT' | 'INFO' | 'assembly' | 'contig' | 'SAMPLE' | 'PEDIGREE' | 'pedigreeDB') ) >token_begin @token_middle %meta_typeid_generic ;
+    meta_key        = ( (alnum | '.' | '_' | '-' )+ - (punct)+ ) >token_begin @token_middle %meta_key_generic;
     
     ## A value can be expressed in multiple ways:
     ## - The line is a single key-value pair (meta_value): the value cannot start with an angle bracket, quote or linebreak, but it is free text thereafter
     ## - One of multiple key-value pairs, non-quoted (meta_field_value): the value cannot contain a comma -end of field-, or a closing angle bracket -end of entry-, or quote
     ## - One of multiple key-value pairs, quoted (meta_field_desc): the value can contain any character except for non-escaped quotes
-    meta_value      = ((print - ("<" | "\"" | NL)) (print)*)	>token_begin $token_middle %token_end ;
-    meta_field_value= (print - ("," | ">" | "\""))+		>token_begin $token_middle %token_end ;
-    meta_field_desc = (("\\" print) | (print - "\""))*		>token_begin $token_middle %token_end ;
+    meta_value      = ((print - ('<' | '"' | NL)) (print)*)	>token_begin $token_middle %token_end ;
+    meta_field_value= (print - (',' | '>' | '"'))+		>token_begin $token_middle %token_end ;
+    meta_field_desc = (('\\' print) | (print - '"'))*		>token_begin $token_middle %token_end ;
```

Comment:
Yes, I guess we do.
Passed: https://github.com/EBIvariation/vcf-validator/blob/develop/test/input_files/v4.3/passed/passed_meta_format.vcf#L3
Failed: https://github.com/EBIvariation/vcf-validator/blob/develop/test/input_files/v4.2/failed/failed_meta_001.vcf#L3